src/tools/rbd/action/Import.cc

```diff
@@ -335,7 +337,8 @@ static int read_tag(int fd, __u8 end_tag, int format, __u8 *tag, uint64_t *readl
   return 0;
 }
 
-int do_import_diff_fd(librbd::Image &image, int fd, bool no_progress, int format, size_t sparse_size)
+int do_import_diff_fd(librados::Rados &rados, librbd::Image &image, int fd,
```

Comment:
Has `rados` been added to check for `rbd_skip_partial_discard` config setting? It still uses global for this.